pkg/didcomm/protocol/middleware/presentproof/middlewares.go

```diff
@@ -56,6 +61,13 @@ func SavePresentation(p Provider) presentproof.Middleware {
 			}
 
 			var names []string
+			var properties = metadata.Properties()
+
+			// nolint: errcheck
+			myDID, _ := properties[myDIDKey].(string)
```

Comment:
why are we ignoring the error?

Reply:
It is boolean that says whether we are able to assert value to a string. By default, it will be an empty string. The is no need to handle it.

Reply:
@soluchok correct but, would that not mean there's a bug elsewhere?

Reply:
It can be a bug. I will fix it.